Cargo.toml

```diff
@@ -63,23 +63,23 @@ abstract-app = { path = "packages/abstract-app" }
 
 # Keep these as path, creates cirular dependency otherwise
 # Only need to re-publish all contracts if a re-publish of abstract-boot is required
-abstract-boot = {path = "packages/abstract-boot" }
+abstract-boot = { path = "packages/abstract-boot", version = "0.13.2" }
```

Comment:
why are we setting a version here?

Reply:
Needed a version for publishing... will remove.